specs/dwritecore/DWriteCore.md

```diff
@@ -510,9 +510,189 @@ The following other DirectWrite APIs also use the hybrid font selection algorith
 -   `IDWriteTextLayout` if the typographic font family model is enabled by calling
     `IDWriteTextLayout4::SetFontAxisValues` or `IDWriteTextLayout4::SetAutomaticFontAxes`
 
-# Examples
-
-## Creating an isolated factory
+## Color Font Support
+
+By default, a glyph has a shape but no intrinsic color. Both DirectWrite and Direct2D have
+`DrawGlyphRun` methods that render glyph runs by filling the glyph shapes with a specified text
+color. For convenience, we will refer to this as _monochrome_ glyph rendering. All fonts have
+monochrome glyphs, but a color font also has color representations of some glyphs. To render glyphs
+in color, an application must use different glyph rendering APIs instead of calling the monochrome
+`DrawGlyphRun` method.
+
+Note that color font support does not affect text layout it at all. The text layout always produces
```

Comment:
What exactly is 'text layout' in this context? Another separate function of DWrite? Could we be a little more specific here, perhaps referencing the relevant APIs?

Reply:
_Text layout_ is the process of converting Unicode text + formatting information to a sequence of glyph runs. DWrite has a text layout API (IDWriteTextLayout), but applications can also implement their own text layout. _Text rendering_ takes the glyph runs produced by text layout and converts them to pixels.

Reply:
I see, so when you say "the text layout always produces monochrome glyph runs", do you mean IDWriteTextLayout always produces them?

Reply:
I've added some additional text to hopefully clarify this.